### DISCUSSION TITLE
Tomzn TOMPD-63WIFI (ETU9-IOT-WIFI) configuration file

### DISCUSSION THREAD

**Discussion by AlexK98** [make-all/tuya-local#2579] [General]
### Tomzn TOMPD-63WIFI (ETU9-IOT-WIFI)

Here is a configuration file for device mentioned in the topic. Matches 100% as of Tuya-Local v2024.11.3.
As I'm very much new to HA topic and hope you can improve on what I've done. :)

[tompd_63wifi_breaker.zip](https://github.com/user-attachments/files/17966689/tompd_63wifi_breaker.zip)

`2024-11-30 02:32:38.023 WARNING (MainThread)
[custom_components.tuya_local.config_flow] Device matches tompd_63wifi_breaker with quality of 100%. DPS: { "updated_at": 1732926753.9392996, "1": 0, "9": 0, "11": false, "12": false, "13": 0, "14": 0, "15": 0, "16": false, "19": "ETU9-IOT-WIFI", "104": 1, "105": 120}`

Here is how it looks in HA:
![HA-TOMPD-63WIFI](https://github.com/user-attachments/assets/086c1391-0940-45e6-b1e7-7e57931f621d)

The only draw back for now is the lack of being able to define params present in **Alarm Set** menu from **Smart Life App**:

![Alarm Set menu](https://github.com/user-attachments/assets/d5b74ceb-1b07-4c21-92d6-d05aea4493ff)

I figured out that **alarm_set_1** is related to **The Leakage alarm** settings, and **alarm_set_2** is related to **Over Current**, **Over Voltage** and **Under Voltage** alarm settings.

If anyone can help with adding controls to these settings, it would be great. Because these **alarm_set_1** and **alarm_set_2** are present not only on this device.

Cheers.

**Comment by make-all** [comment]
Please submit new files as pull requests.

Discussions are not tracked and it is likely a few weeks before this will get to the front of the queue for proper review, by which time it will be buried and forgotten about.
